todo_app/templates/index.html

```diff
@@ -7,11 +7,42 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+  <div class="row">
+    <div class="col-md-12">
+      <h2>New item</h2>
+      <form action="{{ url_for('add_item') }}" method="POST">
+        <div class="form-group form-row align-items-center">
+          <div class="col-auto">
+            <label for="title-input">Title</label>
+          </div>
+          <div class="col">
+            <input type="text" name="title" id="title-input" class="form-control">
```

Comment:
You could include a `required` attribute so the form won't submit with a blank title